file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      "! This is the position
+      field_position TYPE i,
+      "! <p class="shorttext">Key Field</p>
+      "! This is key field or not
+      iskey          TYPE c LENGTH 1,
+    END OF ty_sbs_segment_fields,
+
+    "! <p class="shorttext">Intelligent Scenario</p>
+    "! This is an intelligent scenario
+    BEGIN OF ty_main,
+      "! $required
+      format_version            TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                    TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Intelligent Scenario</p>
+      "! This is an intelligent scenario
+      "! $required
+      intelligent_scenario      TYPE ty_intelligent_scenario,
+      "! <p class="shorttext">Objects Based on Scenario</p>
+      "! These are the objects based on scenario
+      scenario_ddl_objects      TYPE STANDARD TABLE OF ty_scenario_ddl_objects WITH DEFAULT KEY,
+      "! <p class="shorttext">Binding Mappings</p>
+      "! These are scenario binding mappings
+      binding_mappings          TYPE STANDARD TABLE OF ty_binding_mappings WITH DEFAULT KEY,
+      "! <p class="shorttext">Dataset Bindings</p>
+      "! These are scenario dataset bindings
+      bindings                  TYPE STANDARD TABLE OF ty_bindings WITH DEFAULT KEY,
+      "! <p class="shorttext">Signatures</p>
+      "! These are the scenario signatures
+      signatures                TYPE STANDARD TABLE OF ty_signatures WITH DEFAULT KEY,
+      "! <p class="shorttext">Signature Fields</p>
+      "! These are the fields of signature
+      signature_fields          TYPE STANDARD TABLE OF ty_signature_fields WITH DEFAULT KEY,
+      "! <p class="shorttext">Modelling Context</p>
+      "! This is the intelligent scenario Modelling context
+      modelling_context         TYPE STANDARD TABLE OF ty_modelling_context WITH DEFAULT KEY,
+      "! <p class="shorttext">Modeling Context Filters</p>
+      "! These are the modeling context filters
+      modelling_context_filters TYPE STANDARD TABLE OF ty_modelling_context_filters WITH DEFAULT KEY,
+      "! <p class="shorttext">Parameter Defaults</p>
+      "! These are the scenario parameter defaults
+      parameter_defaults        TYPE STANDARD TABLE OF ty_parameter_defaults WITH DEFAULT KEY,
+      "! <p class="shorttext">Parameter Signature</p>
+      "! These are the scenario parameters signature
+      parameters                TYPE STANDARD TABLE OF ty_parameters WITH DEFAULT KEY,
+      "! <p class="shorttext">Additional Information</p>
+      "! These are the scenario additional information
+      additional_info           TYPE STANDARD TABLE OF ty_additional_info WITH DEFAULT KEY,
+      "! <p class="shorttext">Additional Information Texts</p>
+      "! These are the scenario additional information texts
+      additional_info_text      TYPE STANDARD TABLE OF ty_additional_info_text WITH DEFAULT KEY,
```

Comment:
What is the difference between `additional_info` and `additional_info_text`? Is the latter the text table of the first? If so, I suggest to combine the text into `additional_info`.

AFF structures don't need to represent DB table structure ;)

Reply:
This structure is merged.